### PR TITLE
Bug fixed.

### PR DESCRIPTION
**Pull request by fabrisiocm**
Fixed two lines for the ExecuteNonQuery function.

### CODE REVIEW

File: ulacitbnb/Controllers/PaymentController.cs [hunk 1]
```diff
@@ -135,6 +135,7 @@ public IHttpActionResult Enter(Payment payment)
                     sqlCommand.Parameters.AddWithValue("@Pay_Total", payment.Pay_Total);
 
                     sqlConnection.Open();
+                    int rowsAfected = sqlCommand.ExecuteNonQuery();
```
Comment (dlamli) [inline ulacitbnb/Controllers/PaymentController.cs]: Change rowsAfected to "rowsAffected".

File: ulacitbnb/Controllers/PaymentController.cs [hunk 2]
```diff
@@ -210,6 +211,8 @@ public IHttpActionResult Delete(int id)
 
                     sqlCommand.Parameters.AddWithValue("@Pay_ID", id);
                     sqlConnection.Open();
+
+                    int rowsAfected = sqlCommand.ExecuteNonQuery();
```
Comment (dlamli) [inline ulacitbnb/Controllers/PaymentController.cs]: Change rowsAfected to "rowsAffected".

File: ulacitbnb/Controllers/ReservationController.cs [hunk 3]
```diff
@@ -226,6 +227,8 @@ public IHttpActionResult Delete(int id)
 
                     sqlCommand.Parameters.AddWithValue("@Res_ID", id);
                     sqlConnection.Open();
+
+                    int rowsAfected = sqlCommand.ExecuteNonQuery();
```
Comment (dlamli) [inline ulacitbnb/Controllers/ReservationController.cs]: Change rowsAfected to "rowsAffected".

File: ulacitbnb/Controllers/PaymentController.cs [hunk 4]
```diff
@@ -135,6 +135,7 @@ public IHttpActionResult Enter(Payment payment)
                     sqlCommand.Parameters.AddWithValue("@Pay_Total", payment.Pay_Total);
 
                     sqlConnection.Open();
+                    int rowsAffected = sqlCommand.ExecuteNonQuery();
```
Comment (Gabuardi) [inline ulacitbnb/Controllers/PaymentController.cs]: There is no need to create a variable, just do this

```suggestion
                    sqlCommand.ExecuteNonQuery();
```

**Review by Gabuardi — CHANGES_REQUESTED** [review]

**Review by Gabuardi — APPROVED** [review]

**Pull request merged**